content/en/blog/plugins-early-access.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Falco Plugins Early Access
+date: 2021-10-12
+author: Mark Stemm
+slug: plugins-early-access
+---
+
+One upcoming feature in Falco that we're really excited about is the ability to extend Falco's functionality via the use of [plugins](https://deploy-preview-493--falcosecurity.netlify.app/docs/plugins/). We'll be demoing this capability during Kubecon North America 2021. Specifically, we'll be showing off support for [AWS Cloudtrail](https://aws.amazon.com/cloudtrail/) logs in Falco, with a [Cloudtrail](https://github.com/falcosecurity/plugins/tree/master/plugins/cloudtrail) plugin that can read cloudtrail logs and return them as events and new rules that look for suspicious/notable activity in cloudtrail logs.
```

Comment:
The plugin doesn't provide the rules, so I changed the above a bit to say "plugin and related falco rules that can do the following", but otherwise sure.